server/schema.py

```diff
@@ -30,6 +27,15 @@ class Meta:
         interfaces = (graphene.relay.Node,)
 
 
+class Item(SQLAlchemyObjectType):
+    stats = graphene.List(ItemStats)  # Use list instead of connection

```

Comment:
Prefer to use a list instead of a connection since I don't expect these should ever be paginated, so a connection just adds unnecessary complexity.